roadmap.qmd

```diff
@@ -1,158 +1,215 @@
-
-: List of documentation deliverables with approximate timelines for
-uploading to an archive for the first time.
-
-### Training and education
-
-| Status | Timeline | Deliverable | Description |
-|------------------|------------------|-------------------|-------------------|
-| {{< var status.planned >}} | TBD | Tutorial | Installing and setting up Seedcase on a server so Data Resources can be created |
-| {{< var status.planned >}} | TBD | Tutorial | Using Seedcase to create and manage a Data Resource |
-| {{< var status.planned >}} | TBD | Tutorial | Requesting to start a data project for a Data Resource |
-
-: List of training and education deliverables with approximate timelines
+#### Tutorials
+
+Tutorials are learning-oriented documentation that includes multi-day
+courses, online tutorials, and learning shorts. These will be developed
+as [open educational
+resources](https://en.wikipedia.org/wiki/Open_educational_resources).
+
+| Status | Learning resource |
+|-----------------------------------------------|-------------------------|
+| {{< var status.wip >}} | [`rostools/git4cats`](https://github.com/rostools/git4cats): :cat: Git for Collaborative Teams: An introductory course on using Git and GitHub for team-based collaborative workflows and practices |
+| {{< var status.planned >}} | [`rostools/r-pkg-intro`](https://github.com/rostools/r-pkg-intro): Build research software using R as a solo developer in research environments. |
+| {{< var status.potential >}} | Course: Iterative project management and development for effective collaboration and work. |
+| {{< var status.potential >}} | Course: Introduction to building research software using Python as a solo developer in research environments. |
+| {{< var status.potential >}} | Course: Building research software in a team. |
+
+: "Wishlist" of learning-type documentation products we are currently
+working on, will work on, or hope to work on.
+
+#### How-to guides
+
+How-to guides are goal-oriented documentation that helps the reader get
+something specific done. These include how-to guides, usage guides, and
+user manuals.
+
+| Status | Guides or manuals |
+|-----------------------------------------------|-------------------------|
+| {{< var status.wip >}} | [`sprout.seedcase-project.org`](https://sprout.seedcase-project.org/docs/guide) usage guide: Using and interacting with `seedcase-sprout`. |
+| {{< var status.planned >}} | Flower usage guide: Using and interacting with `seedcase-flower`. |
+| {{< var status.planned >}} | Propagate usage guide: Using and interacting with `seedcase-propagate`. |
+| {{< var status.planned >}} | Garden usage guide: Using and interacting with `seedcase-garden`. |
+| {{< var status.planned >}} | Sprout data extensions usage guide: Installing or creating extensions for `seedcase-sprout`. |
+| {{< var status.planned >}} | Installing and setting up Seedcase software on server environments. |
```

Comment:
Remove -- installing on server envs will be a part of the installation guides for the individual apps anyway